cmd/config/create.go

```diff
@@ -0,0 +1,46 @@
+package config
+
+import (
+	"github.com/screwdriver-cd/sd-local/config"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var (
+	configNew = config.New
+)
+
+func newConfigCreateCmd() *cobra.Command {
+	configCreateCmd := &cobra.Command{
+		Use:   "create [name]",
+		Short: "Create the config of sd-local",
+		Long: `Create the config of sd-local.
+The new config has only launcher-version and launcher-image.`,
+		Args: cobra.ExactArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			name := args[0]
+
+			path, err := filePath()
+			if err != nil {
+				logrus.Fatal(err)
+			}
+
+			configList, err := configNew(path)
```

Comment:
is it better to use config instead of configList?

Reply:
Thx. I fixed it.